lib/parse.js

```diff
@@ -1,18 +1,42 @@
-export default function parse(fileContent: String): Object {
-  // For some reason, we can't `import` babylon using an es6 import.
+const TYPESCRIPT_FILE_PATH_REGEX = /\.tsx?$/
+
+export default function parse(fileContent: String, absolutePathToFile: String): Object {
```

Comment:
Not the cleanest function signature (adding `absolutePathToFile: String`). I'm too lazy to come up with a better solution. Ideally I feel like this module should not incorporate any concept of files at all.

Reply:
Agreed. I wonder if this would be better as an options object with a key of `typePlugins`?

Reply:
That's exactly what I thought as well, and is what I did initially. However, I opted for the more rudimentary solution as I feel an options object parameter just introduces bloat and will make things harder to refactor. I'm up for either though 🤷‍♂️